frontend/service/oauth.service.js

```diff
@@ -1,7 +1,10 @@
+const jwt = require('jsonwebtoken')
+
 export class OAuthService {
-  constructor (store, config) {
+  constructor (store, config, app) {
```

Comment:
Is this correct? the wikibase service is not providing the `app` parameter: https://github.com/PhiloBiblon/philobiblon-ui/blob/master/frontend/service/wikibase.service.js#L26
I think that we can modify the constructor in this way:
```
  constructor (store, app) {
    this.$store = store
    this.$config = app.$config
    this.$cookies = app.$cookies
  }
```
and modify the wikibase service to pass `app` instead of `config`.